### DISCUSSION TITLE
Converting to Class Components ❘ 195

### DISCUSSION THREAD

**Discussion by Z-Farook** [chrisminnick/react-js-foundations#5] [Q&A]
Hi @chrisminnick , here you are trying to do what is described on the page on page `195`

https://github.com/chrisminnick/react-js-foundations/blob/e949503c05b6d0f21e9e3ee3323638f775c408d9/book-code-listings/chapter06/RemindersClass/InputForm.js#L3      

`" `
Since all of the state of our application lives in the App component, converting the other components
is straightforward and simple. I’ll show how to convert the first InputForm to a class, and then the
same steps can be followed to convert the others:
1. Import Component from react at the very beginning of the file:
import {Component} from 'react';
2. Replace the function header with a class header:
class InputForm extends Component {
3. Wrap the event handler functions and the return statement with the render method and
change references to props to references to this.props:
....
....
`"`
But this is not correct here `RemindersClass`, as you are still using/showing the function component here!! 
Can you fix it, please?

**Comment by chrisminnick** [comment]
Hi @Z-Farook,

Thanks for writing and for letting me know about this. This is fixed now.

Best,
-Chris
